Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -69,11 +69,15 @@ class RemoteTabsPanel: UITableViewController, HomePanel {
         self.profile.getClientsAndTabs().upon({ tabs in
             if let tabs = tabs.successValue {
                 log.info("\(tabs.count) tabs fetched.")
-                self.clientAndTabs = tabs
+                self.clientAndTabs = tabs.filter({$0.tabs.count > 0})
```

Comment:
Nit: Don't need the ( ) brackets unless we want to keep them in as per our styleguide?